js/tests/enhancedDate.spec.js

```diff
@@ -0,0 +1,82 @@
+//suite description
+describe("Enhanced Date", function (){
+var testDate = enhancedDate.setDate(new Date(2015, 5, 30)); 
+    
+//  var enhancedDate;
+//  
+//  beforeEach(function () {
+//      enhancedDate = new enhancedDate();
+//  });
+
+//Get Day Name Test  
+  describe("Get Day Name", function(){
+    
+      it("should be a string and should return Tuesday", function(){
+          expect(enhancedDate.getDayName(testDate)).to.be.a("string").and.to.equal("Tuesday");
+      });
+  });
+
+
+//Get Month Name Test  
+  describe("Get Month Name", function(){
+    
+      it("should be a string and should return June", function(){
+          expect(enhancedDate.getMonthName(testDate)).to.be.a("string").and.to.equal("June");
+      });
+  });
+
+
+//Is Today Test  
+  describe("Is Today", function(){
+    
+      it("testDate should not be a string and should return false ", function(){
+          expect(enhancedDate.isToday(testDate)).to.not.be.an("string").and.to.not.be.true;
+      });
+    
+//      it("should not be a string ", function(){
+//          expect(enhancedDate.isToday(testDate)).to.not.be.an("string");
+//      });
+        
+  });
+
+
+//Is Future Test 
+  describe("Is Future", function(){ 
+
+      it("testDate should return true", function(){
+          expect(enhancedDate.isFuture(testDate)).to.be.true;
+      });
+    
+      it("should not be a string ", function(){
```

Comment:
I tried doing that, but I kept getting errors when I tried to create a test
date for today and a test date for a past date, and errors if I used an
empty function to test for today.  Not sure why, so I went with these for
now.

On Sun, May 17, 2015 at 10:21 PM, jamdon <notifications@github.com> wrote:

> In js/tests/enhancedDate.spec.js
> <https://github.com/JSCRIPT300-Spring2015/Module_6/pull/2#discussion_r30482259>
> :
>
> > +
> > +//      it("should not be a string ", function(){
> > +//          expect(enhancedDate.isToday(testDate)).to.not.be.an("string");
> > +//      });
> > +
> > +  });
> > +
> > +
> > +//Is Future Test
> > +  describe("Is Future", function(){
> > +
> > +      it("testDate should return true", function(){
> > +          expect(enhancedDate.isFuture(testDate)).to.be.true;
> > +      });
> > +
> > +      it("should not be a string ", function(){
>
> A better second test here would be to use another date to test the
> function returns false under appropriate conditions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JSCRIPT300-Spring2015/Module_6/pull/2/files#r30482259>
> .
>